modules/oaipmh-persistence/src/test/java/org/opencastproject/oaipmh/persistence/impl/OaiPmhPersistenceTest.java

```diff
@@ -239,4 +221,49 @@ public void testLimitOffset() throws Exception {
     Assert.assertEquals(1, search.size());
     Assert.assertEquals(mp2.getIdentifier().toString(), search.getItems().get(0).getId());
   }
+
+  @Test
+  public void tesRemovalOfOrphanedElements() throws Exception {
+    oaiPmhDatabase.store(mp1, REPOSITORY_ID_1);
```

Comment:
Small nitpicking comment: There's a t missing in the method name -> tes**t**RemovalOfOrphanedElements

Reply:
Done